game/collide.cpp

```diff
@@ -191,30 +212,463 @@ void __cdecl GetNewRoom(int x, int y, int z, __int16 roomID) {
 	DrawRoomsArray[DrawRoomsCount++] = roomID;
 }
 
+void __cdecl ShiftItem(ITEM_INFO* item, COLL_INFO* coll) {
```

Comment:
Pointers on the right side:

`void __cdecl ShiftItem(ITEM_INFO *item, COLL_INFO *coll) {`

Reply:
fixed